core/Cargo.toml

```diff
@@ -28,7 +29,7 @@ serde_yaml = "0.8"
 uuid = { version = "0", optional = true }
 
 [features]
-actix = ["v2", "actix-http", "actix-web"]
+actix = ["v2", "actix-http", "actix-web", "actix-multipart"]
```

Comment:
Let's have a `multipart` feature instead, which depends on `actix-multipart` please? Like, how we have it for `chrono` and `datetime` (the dependency should be opt-in).